google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelector.java

```diff
@@ -391,6 +393,25 @@ public void showPopup(RelativePoint showTarget) {
     }
   }
 
+  @Override
+  public void addFocusListener() {
+    super.addFocusListener();
+
+    getTextField().addFocusListener(new FocusListener() {
+      @Override
+      public void focusGained(FocusEvent e) {
+        // do nothing
+      }
+
+      @Override
+      public void focusLost(FocusEvent event) {
+        if (!event.isTemporary()) {
```

Comment:
a lose focus event is permanent if it remains unfocused after everything is done. 

I noticed that this event was getting fired twice always: once with temporary=true and once with temporary=false. I did a little digging and saw that we can ignore the temporary once.